docs/features/overrides.md

```diff
@@ -208,4 +209,53 @@ spec:
                   - name: extra-volume
                     configMap:
                       name: extra-config
+```
+
+### Example: Add an environment variable from secretRef to the frontend pod
+
+```yaml
+apiVersion: temporal.io/v1beta1
+kind: TemporalCluster
+metadata:
+  name: prod
+spec:
+  # [...]
+  services:
+    frontend:
+      overrides:
+        deployment:
+          spec:
+            template:
+              spec:
+                containers:
+                  - name: service
+                    envFrom:
+                      - secretRef:
+                          name: frontend
+```
+
+## Overrides container resources all services
```

Comment:
Could you please move this section to `Overrides for all services` and rename the title to something like `Example: Override containers resources` ?

Reply:
sure